### ISSUE TITLE
[Bug] Blazor redirect to Azure AD B2C sign-in page

### ISSUE THREAD

**Issue by frindler** [AzureAD/microsoft-identity-web#1727]
My Blazor Server app is linked to Azure AD B2C and I can sign in without issues by putting in a

`<a href="MicrosoftIdentity/Account/SignIn">Sign in</a>`

However, if I try to do this programmatically, the following does not work:

`navigationManager.NavigateTo("MicrosoftIdentity/Account/SignIn");`

(I have done `@inject NavigationManager navigationManager` in the Blazor component).

I just get the standard "Sorry, there's nothing at this address.".

However, if I then just go to the browser nav bar I see "https://localhost:7258/MicrosoftIdentity/Account/SignIn" and when I press Enter it goes to the Azure AD sign-in as required.

**Comment by frindler** [comment]
OK, fixed it: `navigationManager.NavigateTo("MicrosoftIdentity/Account/SignIn", forceLoad: true);` does the job.

**Comment by naesevol** [comment]
I'm also experiencing this. And this "workaround" isn't great. I am flashed with a "Sorry there is nothing found at this address." Page before the forceReload kicks in and does the work to route me to the sign-up/sign-in page.

This is pretty bad as far as UX goes. Would love a proper fix.
To be clear, this is happening when I turn off 

`options.FallbackPolicy = options.DefaultPolicy;`

And I have to turn that off in order to get onClick events to work on anonymous pages.

**Comment by jvanlangen** [comment]
I'm experiencing the same problem. I copied the `<AuthorizeView>` from an example, where it directly redirects to the microsoft login page.

I copied the block to my existing blazor server application, and the `Login` link is rendered. When pressing that link the "Sorry, there's nothing at this address." shows up..

Rather not create "code-behind" I think there is something missing which translates the MicrosoftIdentity to a valid url?
